src/main/java/hw_8/Task_1/Application.java

```diff
@@ -0,0 +1,74 @@
+package hw_8.Task_1;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Map<String, Person> personMap = new HashMap<>();
+        Person onePerson = new Person(25, "Oleg", "Vahnenko");
+        Person twoPerson = new Person(35, "Nana", "Petroff");
+        personMap.put("AAAA", onePerson);
+        personMap.put("MMM", twoPerson);
+        System.out.println("Enter command");
+        String addInformation = scanner.nextLine();
+        System.out.println("Enter key");
+        while (true) {
+            String keySurname = scanner.nextLine().toUpperCase();
+            if ("addPerson".equals(addInformation)) {
+                addPerson(personMap, scanner, keySurname);
+            } else if ("getPerson".equals(addInformation)) {
+                getPerson(personMap, keySurname);
+
+            } else if ("Exit".equals(addInformation)) {
+                System.out.println("Program close");
+            } else
+                System.out.println("Wrong command");
+            break;
```

Comment:
you should exit the application only entering "exit" command, but in fact if I enter "abracadabra" app is finisged as well. So this breal is should be removed and just leave 
} else {
                System.out.println("Wrong command");
            }